File: Aula4/Exercicios-aula4/projeto_circ_cil/src/main/java/aula4/projeto_circ_cil/Circulo.java
package aula4.projeto_circ_cil;
import javax.swing.JOptionPane;


public class Circulo {
    
    public float raio;
    public final float pi = 3.1415f;
    
    public Circulo(){}
    public Circulo(float raio){
        this.raio = raio;
    }
    
    public float calculaArea(){
        return pi * (raio*raio);
    }
    
    public void imprimeDados(){
        if(raio > 0.0f){
        JOptionPane.showMessageDialog(null,
                "Raio do círculo: " + 
                        this.raio + 
                        " área: " + 
                        calculaArea());
        }      
    }
    
}

File: CsharpStyle/Aula4/Ex1/classes/Circulo.cs
namespace Ex1;

public class Circulo
{
    public float Raio;
    public float Pi = 3.1415f;

    public Circulo() { }
    public Circulo(float raio)
    {
        Raio = raio;
    }

    public float calculaArea()
    {
        return Pi * (Raio * Raio);
    }

    public override string ToString()
    {
        if (Raio > 0.0f)
        {
            return $"Raio do circulo: {Raio.ToString()}, área: {calculaArea().ToString()}";
        }
        return "Raio inválido";
    }
}
